packages/editor/src/components/index.js

```diff
@@ -33,6 +33,7 @@ export { default as PostLastRevisionCheck } from './post-last-revision/check';
 export { default as PostLockedModal } from './post-locked-modal';
 export { default as PostPendingStatus } from './post-pending-status';
 export { default as PostPendingStatusCheck } from './post-pending-status/check';
+export { default as PostPermalink } from './post-permalink';
```

Comment:
Thank you for your contribution @igmoweb 👍 It seems fine to expose this component. Would it be possible to add a changelog entry specifying that this component is now exposed (new features section)?

Reply:
@jorgefilipecosta Added. I hope it's fine this way.